bot/bot_swedish_pipeline.yml

```diff
@@ -24,18 +24,29 @@ sfi_perm_age:
 
 sfi_perm_personalnumber:
   node_type: "SimpleOptionNode"
-  content: "Have you studied swedish before?"
+  content: "Do you have a personal number?"
   exit_node_content: "Back to SFI"
   exit_node_id: "head_topic_swedish"
   options:
     - - content: "Yes"
-        next_node_id: "sfi_perm_levelb1"
+        next_node_id: "sfi_level_swedishbefore"
       - content: "No"
         next_node_id: "sfi_perm_no"
 
+sfi_level_swedishbefore:
```

Comment:
Could you replace `level` to `perm`? Other ids have names as `sfi_perm_*`

Reply:
done